plugins/woocommerce/tests/php/src/Internal/Admin/BlockTemplates/BlockBasedTemplateTest.php

```diff
@@ -0,0 +1,367 @@
+<?php
```

Comment:
This file can be renamed to `BlockTemplateTest`.

Reply:
Done